setup.py

```diff
@@ -3,12 +3,12 @@
 setup(
     name='ipam-client',
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
-    version='0.3.3',
+    version='0.3.4',
```

Comment:
Please avoid changing version in a feature commit, to make release / rollback easier.